install/third-party/rmi/install.yml

```diff
@@ -0,0 +1,14 @@
+id: third-party-newrelic-java-rmi
```

Comment:
We can remove this install plan as these are now deprecated in favor of data sources:
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans-deprecated

Reply:
Removed Install Plan and its references from Quickstart.